### DISCUSSION TITLE
Need help with Chrome license

### DISCUSSION THREAD

**Discussion by WesleyDellth** [aclap-dev/video-downloadhelper#1051] [License & Purchase requests]
### What's your issue?

It happened that i bought chrome version where it doesnt allow to download vdo from youtube.
Then i just read that it is downable via firefox extension only.
My question is, Is it possible to swap my license from Chrome to firefox instead?
Or if the refund is possible. I prefer to get firefox license more.

### The 10 first digits (only 10) of your license number or email address

6194601182

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
